job-cleanup/Dockerfile

```diff
@@ -0,0 +1,9 @@
+from alpine:3.5
+
+arg CLIENT_TARBALL=https://github.com/kubernetes-incubator/client-python/archive/master.zip
```

Comment:
2.0.0a1 (the latest tagged version) is missing some options I'd been trying for cascading deletion. That being said, we can probably switch back to it since cascading delete is totally broken server-side even with library support